packages/hint-doctype/README.md

```diff
@@ -0,0 +1,114 @@
+# Check that the DOCTYPE tag of the document is valid
+
+This hint checks if the HTML is using the most modern DOCTYPE.
+
+## Why is this important?
+
+> In HTML, the doctype is the required "<!DOCTYPE html>" preamble
+found at the top of all documents. Its sole purpose is to prevent
+a browser from switching into so-called “quirks mode” when rendering
+a document; that is, the "<!DOCTYPE html>" doctype ensures that the
+browser makes a best-effort attempt at following the relevant specifications,
+rather than using a different rendering mode that is incompatible
+with some specifications.
+
+***From [MDN glossary for DOCTYPE specification][docmdn].***
+
+## What does the hint check?
+
+This hint checks if the HTML is using the most modern and valid DOCTYPE.
+
+It checks that the DOCTYPE is in the first line and that there are no other
```

Comment:
> It checks that the DOCTYPE is in the first line and that there are no other
lines before the DOCTYPE.

If there are only whitespace characters before the doctype it's fine and everything should work as expected.

The problem with enforcing this is that some back-end languages generate HTML with a lot of space (from experience I know Java and PHP do this - e.g.: Look at the source code for: https://www.java.com/en/).

Reply:
Cc @webhintio/contributors

Reply:
@CKGrafico @isabellachen Can you also change the code for this? Thanks!